packages/syft/src/syft/grid/duet/__init__.py

```diff
@@ -296,3 +299,66 @@ def join_duet(
     # begin_duet_client_logger(duet.node)
 
     return duet
+
+
+def test_duet_network(loopback: bool = False) -> None:
+    if not loopback:
```

Comment:
loopback=True still uses the same signalling server it just exchanges the IDs automatically using a file, so we should still check all of these main url service checks. There is the case where a user can run their own signalling server so it might be nice to be able to pass that in as a parameter and have these checks run against it to make sure that the python process can read the url on the port you supply.

like:
```
sy.test_duet_network(network_url="http://192.168.0.2:5432")
```

Reply:
Not sure how I missed those 😞️. Fixed.